BREAKING.md

```diff
@@ -17,6 +17,26 @@ It's important to communicate breaking changes to our stakeholders. To write a g
 
 # 2.0.0-internal.4.1.0
 
+## 2.0.0-internal.4.1.0 Breaking changes
+
+-   [Ability to enable grouped batching](#Ability-to-enable-grouped-batching)
+
+### Ability to enable grouped batching
```

Comment:
@tylerbutler - I'd think that we would enable changeset thing once we have a flow that auto-generates breaking.md, i.e. developers need to put text only in one place. Current state seems very inconvenient.

Reply:
Will do this in a future PR as it's just a name change (don't want to bloat this PR more)